spec/requests/users_spec.rb

```diff
@@ -1,7 +1,20 @@
 require 'rails_helper'
 
-RSpec.describe 'Users', type: :request do
-  describe 'GET /index' do
-    pending "add some examples (or delete) #{__FILE__}"
+RSpec.describe 'Posts', type: :request do
+  describe 'GET /users' do
+    it 'renders the index template' do
+      get '/users'
+      expect(response).to have_http_status(:ok)
+      expect(response).to render_template(:index)
+      expect(response.body).to include('List of all Users')
+    end
+  end
+
+  describe 'GET /users/:user_id' do
+    it 'renders the users by id' do
+      get '/users/74'
+      expect(response).to have_http_status(:ok)
+      expect(response).to render_template(:show)
+    end
   end
```

Comment:
@zieeco Hi, thanks for the review. As you can see from the error message, it couldn't find the user with id 74, it requires an user with id 74 because it's checking if the user has the posts. Check my screenshot, all the test cases are passing because I have the user with id 74 in my database.
![Screenshot from 2023-07-19 22-23-44](https://github.com/sumon766/techbright/assets/22003499/0cbdc920-a936-49a2-b659-1ab32511333e)